### DISCUSSION TITLE
images in RST files?

### DISCUSSION THREAD

**Discussion by finite9** [getpelican/pelican#3070] [General]
Can I add images to restructuredtext or is that only for markdown files?

I tried with:
`.. image:: ../static/yi_jing_01_chien.jpg
   :height: 100 %
   :width: 100 %
   :scale: 10 cm
   :alt: alternate text
   :align: right`

Although Pelican with Elegant theme did not like the % sign, or scale attribute.

Which I pulled from some site about restrucyured text, but it doesn't view the image in the generated output/site.  Just a placeholder icon.

**Comment by finite9** [comment]
figured it out... it would have been logical to assume the root of the site /var/www/myblog was the starting folder.  Turns out that "content" is the starting root so when adding images I had to use
`.. image:: images/image.jpg`

where the full path was /var/www/myblog/content/images.  Now it works :)

**Comment by mpounsett** [comment]
If you look at the section on [linking to static files](https://docs.getpelican.com/en/latest/content.html#linking-to-static-files) you'll find a mechanism for doing this.

```rst
.. image:: {static}/images/yi_jing_01_chien.jpg
   :height: 100%
   :width: 100%
   :scale: 10cm
   :alt: alternate text
   :align: right
```
